flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlMapType.java

```diff
@@ -25,7 +25,7 @@
 import org.apache.calcite.sql.type.SqlTypeName;
 
 /**
- * Extended Flink MapType.
+ * Parse column of Map type.
```

Comment:
I think we should move this class out of `ddl` package, as well as other flink defined types.

Reply:
I agree, moved to package `org.apache.flink.sql.parser.type`